src/backend/database_migrations/versions/20220524_174911_add_auth0_org_id_column_to_groups_table.py

```diff
@@ -0,0 +1,35 @@
+"""Add auth0 org id column to groups table
```

Comment:
This file being in this PR is an oopsie, but incredibly minor and can be ignored.

Since the analytics stuff is a long-lived feature branch, everything should really base off `vince/analytics` branch, with that branch directly getting merges from `trunk` now and then. But muscle memory and I accidentally merged in latest trunk as I was getting ready for PR. But I was going to merge this same stuff directly in `vince/analytics` anyway, so... I'm just going to be lazy and leave it as part of this PR if no one yells at me.